### PR TITLE
Firstb

### PR DESCRIPTION
**Pull request by AnaDenisa**
### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

File: docs/Developers/Writing_Good_Commits.rst [hunk 1]
```diff
@@ -26,6 +26,9 @@ Example of a good commit:
 - `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
    of the issue it closes or fixes.
 
+Use 'Fixes' when you fixed a bug and 'Closes' when your work is 
+related to a new feature.
```
Comment (Mixih) [inline docs/Developers/Writing_Good_Commits.rst]: This needs to be more specific. You should clarify that fixes is for *any* unintended behavior even for developers. (my interpretation of the discussion is that any unintended behavior is a bug whether that is the user or developer).

**Comment by Mixih** [comment]
status/wip

**Comment by Mixih** [comment]
Uh-oh! You have the dreaded merge commit. This is probably because you pushed the rebase(update branch from master) button in github. This is not the proper way of doing this and will break many things. Please do an interactive rebase on the commit before the merge commit and remove the merge commit line(see http://stackoverflow.com/questions/17577409/git-remove-merge-commit-from-history). See https://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing for the proper way to rebase.

**Comment by AnaDenisa** [comment]
how do I find out the sha before the branches deverged for removing the merge commit line? I have written in the terminal git rebase -i master, I wanted to squash the commits :(

**Comment by Mixih** [comment]
`git log` will show you all past commits and their shas

**Pull request merged**